### DISCUSSION TITLE
Courier SMS configuration

### DISCUSSION THREAD

**Discussion by kghost** [ory/kratos#4239] [Q&A]
From doc https://www.ory.sh/docs/kratos/reference/configuration

I wonder what's the difference between `courier.sms` and `courier.channels[id: sms]`

They share same fields, and I tried `courier.sms` which doesn't work at all, but `courier.channels[id: sms]` works fine.

**Comment by vinckr** [comment]
Hey @kghost 
The courier.channels configuration is the current and supported way to configure SMS delivery in Kratos; courier.sms is either deprecated or not officially supported-I am not sure tbh.
